app/src/components/AdminPanel/ConfirmTipsAndArticles/TipRow.tsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable */
+import React, { useState } from "react";
+import { confirmAlert } from "react-confirm-alert";
+import "react-confirm-alert/src/react-confirm-alert.css";
+import { TipItem } from "../../Pages/TipsAndArticles/Tip";
+import ArticleModal from "./confirmArticleModal";
+
+type tipRowProp = {
+  tip: TipItem;
+  confirm: boolean;
+  confirmData: (tipId: string) => void;
+};
+export const TipRow = (props: tipDataRowProp) => {
+  const {tip, confirm } = props;
+  
+  const [openModal, setModalOpen] = useState(false);
+
+  const handleChange = (event) => {
```

Comment:
Please apply the same comments I gave in articleRow. You have many imports, props, and vars that not use. So let's remove them